src/components/calcite-slider/calcite-slider.scss

```diff
@@ -35,6 +35,7 @@
 }
 
 :host {
+  --calcite-slider-internal-animation-timing: calc(250ms * var(--calcite-animation-timing-factor));
```

Comment:
Since there's only a 50ms difference between the internal global vars, could this use either the `medium` or `slow` var? cc @macandcheese

Reply:
Yeah let's standardize to use the established ones here - medium would be the suggestion here.